src/app/REMOVE-chat/core.nim

```diff
@@ -33,7 +33,6 @@ include event_handling
 include signal_handling
 
 proc init*(self: ChatController) =
-  self.handleMailserverEvents()
```

Comment:
@richard-ramos code/files which are currently in any directory containing "REMOVE" in its name are not used/included anywhere in the app, they are here only because of a reference to an old code (how it looked before if we need to check some part) and they will be deleted soon. This is not an issue, just a comment, in any case it does not affect anything.